101-vm-userdata/azuredeploy.json

```diff
@@ -0,0 +1,244 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Storage Account where the Virtual Machine's disks will be placed."
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "User name for the Virtual Machine."
+      }
+    },
+    "userData": {
+      "type": "string",
+      "defaultValue": "echo userData",
+      "metadata": {
+        "description": "String passed down to the Virtual Machine."
```

Comment:
nit: description is kind of vague. maybe Ning has better description than just 'string'

Reply:
Fair point. Talked with Ning and she suggested keeping it as is.